docs/docs/guides/build/assets/asset-selection-syntax.md

```diff
@@ -150,102 +132,116 @@ taxi_zones_file taxi_trips_file
 
 Which would result in the following asset graph:
 
-![Screenshot of Daggy U project graph](/images/guides/build/assets/asset-selection-syntax/select-disjointed-lineages.png)
+![TODO]()
 
 </TabItem>
 </Tabs>
 
----
+## Upstream and downstream layers
 
-### Selecting an asset's entire lineage \{#full-lineage}
+Filtering by upstream and downstream layers can help you understand the data flow within your assets.
 
-To select an asset's entire lineage, add an asterisk (`*`) before and after the asset key in the query.
+- **Upstream assets** provide input to a given asset.
+- **Downstream assets** receive input from a given asset.
+- A **layer** is a level of connection in the graph. One layer up or down means a direct connection, "two layers" means connections two steps away, and so on.
 
-This example selects the entire lineage for the `taxi_zones` asset.
+| Asset selection | Syntax | Description |
+|------|--------|-------------|
+| **All layers upstream of an asset** | `+key:"my_asset"` | Selects all upstream assets that provide input into `my_asset`. |
+| **One layer upstream of an asset** | `1+key:"my_asset"` | Selects assets that directly provide input to `my_asset`. |
+| **Two layers upstream of an asset** | `2+key:"my_asset"` | Selects assets two steps upstream from `my_asset`. |
+| **All layers downstream of an asset** | `key:"my_asset"+` | Selects all downstream assets that depend on `my_asset`. |
+| **One layer downstream of an asset** | `key:"my_asset"+1` | Selects assets that directly receive input from `my_asset`. |
+| **Two layers downstream of an asset** | `key:"my_asset"+2` | Selects assets two steps downstream from `my_asset`. |
+| **One layer upstream and downstream of an asset** | `1+key:"my_asset"+1` | Selects one layer of assets providing input to and receiving input from `my_asset`. |
+| **Two layers upstream and downstream of an asset** | `2+key:"my_asset"+2` | Selects two layers of assets providing input to and receiving input from `my_asset`. |
+| **All layers upstream and downstream of an asset** | `+key:"my_asset"+ ` | Selects all assets upstream and downstream of `my_asset`. |
+
+### Select all upstream layers \{#all-upstream}
+
+To select an asset and all its upstream dependencies, add a plus sign (`+`) before the asset key in the query.
 
-<Tabs>
+This example selects the `manhattan_map` asset and all its upstream dependencies.
+
+<Tabs groupId="examples">
 <TabItem value="python" label="Python">
 
 ```python
-taxi_zones_job = define_asset_job(name="taxi_zones_job", selection="*taxi_zones*")
+manhattan_all_upstream = AssetSelection.from_string("+manhattan_map")
 ```
 
 </TabItem>
 <TabItem value="cli" label="CLI">
 
-When selecting an asset's entire lineage using the CLI, enclose the asterisk (`*`) and the asset key in double quotes (`"`):
+When selecting an asset's dependencies using the CLI, enclose the plus sign (`+`) and the asset key in double quotes (`"`):
```

Comment:
`lineage` is correct because it includes both dependencies and dependents of the asset.